### PR TITLE
Fix support modal broken

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

**Comment by CPatchane** [comment]
[Modales Cassées](https://trello.com/c/BY1BaN7j/739-modales-cass%C3%A9es)

**Comment by ptbrowne** [comment]
I personally prefer when there is a description to a PR, it is easier to understand.

File: src/styles/supportModal.css [hunk 1]
```diff
@@ -2,6 +2,10 @@
   min-height: 8em;
 }
 
+[role=banner] .coz-support-modal {
+  max-width: inherit;
```
Comment (ptbrowne) [inline src/styles/supportModal.css]: inherit ? Why not none here ?

Comment (ptbrowne) [reply src/styles/supportModal.css]: Also I do not understand why everywhere in bar you use [role=banner] and not something like "cozy-bar".

Comment (CPatchane) [reply src/styles/supportModal.css]: You're right, none is a better value 👍 
For the [role="banner"] it's an aria value for the role attribute (https://www.w3.org/TR/wai-aria/#landmark_roles)

Comment (ptbrowne) [reply src/styles/supportModal.css]: Yeah I know it's aria stuff but it still feels weird to me to do this instead of `.cozy-bar`.

**Comment by ptbrowne** [comment]
LGTM

File: src/styles/supportModal.css (hunk 1 above)
Comment (CPatchane) [reply src/styles/supportModal.css]: Since we also use [role=application] for the application part, it's consistent for me.

**Pull request merged**